cves/CVE-2017-17081.yml

```diff
@@ -64,7 +64,14 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  NOTE: This third file was accidentally started - disregard (or don't if you're feeling extra credit-y?)
```

Comment:
```suggestion
  
```

Reply:
We won't count this one toward your grade, although the others are fantastic so I wouldn't worry about it.

Reply:
Noted! I didn't entirely finish the CVE for this one mind you, hence the note.